common-front/lib/store/measurementsStore.ts

```diff
@@ -44,6 +43,9 @@ export const useMeasurementsStore = create<MeasurementsStore>((set, get) => ({
      * @param {Record<string, PacketUpdate>} measurements 
      */
     updateMeasurements: (measurements: Record<string, PacketUpdate>) => {
+
+        const measurementsCopy = get().measurements;
```

Comment:
```suggestion
        const measurementsDraft = get().measurements;
```
I think that name better conveys what this is doing